packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/SubTaskPopup/SubTaskPopup.js

```diff
@@ -24,75 +69,107 @@ function SubTaskPopup(props) {
   } = activeMark
   if (!subTaskVisibility) return null
 
+  const [confirm, setConfirm] = React.useState('pending')
```

Comment:
I'm not 100% on the naming here... maybe `confirmationState`? This is a non-blocker for me though.

Reply:
I like `confirmationState` better 👍